src/sagemaker_containers/worker.py

```diff
@@ -79,14 +77,11 @@ def run(transform_fn, initialize_fn=None, healthcheck_fn=None, module_name=None)
     def invocations_fn():
         transform_spec = transform_fn()
 
-        return Response(response=transform_spec.prediction,
+        return Response(response=transform_spec.serialized_prediction,
```

Comment:
the longer name doesn't seem to add much here

Reply:
I agree.

Reply:
But prediction is miss leading as well. What do you suggest @jesterhazy ?